third_party/terraform/tests/resource_bigtable_instance_test.go

```diff
@@ -190,7 +190,6 @@ resource "google_bigtable_instance" "instance" {
   name = "%s"
   cluster {
     cluster_id   = "%s"
-    zone         = "us-central1-b"
```

Comment:
For posterity: this was removed to make sure the case of defaulting to the provider value works. There are other tests that create explicitly in us-central1-b.